pbs_light.go

```diff
@@ -505,11 +505,13 @@ func sortBidsAddKeywordsMobile(bids pbs.PBSBidSlice, pbs_req *pbs.PBSRequest, pr
 				hbDealIdBidderKey = hbDealIdBidderKey[:min(len(hbDealIdBidderKey), int(pbs_req.MaxKeyLength))]
 				hbSizeBidderKey = hbSizeBidderKey[:min(len(hbSizeBidderKey), int(pbs_req.MaxKeyLength))]
 			}
+
 			pbs_kvs := map[string]string{
```

Comment:
Maybe instead of [this copy](https://github.com/prebid/prebid-server/pull/298/files#diff-444feda808b76687bcc8827c1025876cR541) it would be cleaner here to just make `pbs_kvs` a reference to `bid.AdServerTargeting`, e.g.
```go
// may make sense to just make this an empty map by default instead of having to check for `nil`
if bid.AdServerTargeting == nil {
  bid.AdServerTargeting = make(map[string]string)
}

// In go, [maps are pointers to runtime types](https://dave.cheney.net/2017/04/30/if-a-map-isnt-a-reference-variable-what-is-it), so `pbs_kvs` should act as a reference here
pbs_kvs  = bid.AdServerTargeting

pbs_kvs[hbPbBidderKey]  = roundedCpm
...
```

Reply:
I agree, this suggestion is cleaner. Your call if you want to rewrite it that way in this PR.